src/goto-programs/abstract-interpretation/interval_domain.cpp

```diff
@@ -1120,6 +1121,21 @@ void interval_domaint::assume(const expr2tc &cond)
   }
 #endif
 
+  if(enable_ibex_contractor)
```

Comment:
Add some explanation.

Reply:
Done. 
Added "/// use ibex contractors to reduce the intervals for interval analysis"